src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,116 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { SortOrder } from '../../utils/SortOrder';
+import { SearchLink } from '../SearchLink';
+import { TableLink } from '../TableLink';
+
+type Props = {
+  people: Person[];
+  sort: SortOrder;
```

Comment:
Wrong type for the field `sort`. You expect there column name, but uses type for Sort order which has values asc,desc.